CMakeLists.txt

```diff
@@ -93,7 +93,7 @@ foreach(target_name ${UVC_TARGETS})
 endforeach()
 
 if(BUILD_UVC_SHARED)
-  target_link_libraries(uvc ${LIBUSB_LIBRARIES} ${PTHREAD_LIBRARIES})
+  target_link_libraries(uvc "-L${LIBUSB_LIBRARY_DIRS}" ${LIBUSB_LIBRARIES} ${PTHREAD_LIBRARIES})
```

Comment:
I think we should use either `LIBUSB_LIBRARY_DIRS` or `LIBUSB_LIBRARIES`?
The docs mention `LIBUSB_LIBRARY_NAMES` btw, which seems to be completely outdated.
We should investigate whether out libusb binding actually provides a cmake config (and then use this one) or just use the auto-generated variable names from cmake.